pro/cypress/e2e/step-definitions/signupJourneyCreateOfferer.cy.ts

```diff
@@ -128,9 +127,9 @@ When('I fill identification form with a new address', () => {
   cy.intercept({
     method: 'GET',
     url: '/venue-types',
```

Comment:
Avec un truc comme ça, on peut garder le même nom, et ça reste plus lisible, je pense.

```suggestion
  cy.intercept({
    method: 'GET',
    url: '/venue-types',
    times: 1,
```

Reply:
OK